main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -2353,6 +2339,24 @@ object Resolver {
     // TODO NS-REFACTOR check accessibility
   }
 
+  /**
+   * Finds the struct field that matches the given name `name` in the namespace `ns0`.
+   */
+  private def lookupStructField(name: Name.Label, env: ListMap[String, Resolution], ns0: Name.NName, root: NamedAst.Root): Result[NamedAst.Declaration.StructField, ResolutionError.UndefinedStructField] = {
+    val matches = tryLookupName(Name.mkQName("€" + name.name, name.loc), env, ns0, root) collect {
+      case Resolution.Declaration(s: NamedAst.Declaration.StructField) => s
+    }
+    matches match {
+      // Case 0: No matches. Error.
+      case Nil => Result.Err(ResolutionError.UndefinedStructField(name, name.loc))
+      // Case 1: Exactly one match. Success.
+      case field :: _ => Result.Ok(field)
```

Comment:
This is that bug again right?

Reply:
Oh wait I have a fix. I'll put it in a different PR